src/app/games/hexodia/HexodiaTutorial.ts

```diff
@@ -0,0 +1,101 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { HexodiaState } from './HexodiaState';
+import { HexodiaMove } from './HexodiaMove';
+import { Coord } from 'src/app/jscaip/Coord';
+import { HexodiaConfig, HexodiaRules } from './HexodiaRules';
+import { TutorialStepMessage } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStepMessage';
+import { FourStatePiece } from 'src/app/jscaip/FourStatePiece';
+import { MGPOptional } from '@everyboard/lib';
+
+const _: FourStatePiece = FourStatePiece.EMPTY;
+const O: FourStatePiece = FourStatePiece.ZERO;
+const X: FourStatePiece = FourStatePiece.ONE;
+const N: FourStatePiece = FourStatePiece.UNREACHABLE;
+const defaultConfig: MGPOptional<HexodiaConfig> = HexodiaRules.get().getDefaultRulesConfig();
+const initialState: HexodiaState = HexodiaRules.get().getInitialState(defaultConfig);
+
+export class HexodiaTutorial extends Tutorial {
+
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Initial board and object of the game`,
```

Comment:
"goal of the game" est présent plusieurs, fois, uniformiser pour "object of the game"?